src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -9628,14 +9628,14 @@ void func_80847BA0(GlobalContext* globalCtx, Player* this) {
     }
 }
 
-void func_808486A8(GlobalContext* globalCtx, Player* this) {
-    u8 sp27;
+void Player_SetCamModeAndSeqMode(GlobalContext* globalCtx, Player* this) {
+    u8 seqMode;
```

Comment:
thinking `UpdateCamAndSeqMode` is a bit better. set in the name implies to me that you have control over what it gets set to. While this is just updating it based on many factors (and is directly called by `Player_UpdateCommon`)